### DISCUSSION TITLE
How to unblock Crosh?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1206] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Anybody Know how to open crosh when blocked by policy

**Comment by Luigi-Master** [comment]
I'll look into some exploits for that.

**Comment by 3kh0** [comment, marked as answer]
There is no known way to unblock those. Since crosh is most likely blocked by the policy list, there is no way to bypass it.
